ci-operator/config/openshift/rosa/openshift-rosa-master__e2e.yaml

```diff
@@ -29,6 +29,28 @@ tests:
     test:
     - ref: rosa-test-e2e
     workflow: rosa-test-profile
+- as: rosa-advanced-f3
```

Comment:
Rethinking, `rosa-sts-advanced-f3` could be better for us to separate the jobs from the others when doing a filter on the Prow CI website.